ichnaea/tests/base.py

```diff
@@ -299,6 +300,10 @@ def cleanup_tables(cls, engine):
 
 class RedisIsolation(object):
 
+    def tearDown(self):
+        self.redis_client.flushdb()
+        super(RedisIsolation, self).tearDown()
+
```

Comment:
Oh sorry I see what you mean, you want each subclassing test case to explicitly have to call that out.  I don't like that behaviour, but since it's the existing pattern in the project I'll make that change.